src/runtime/runtime.js

```diff
@@ -29,10 +29,8 @@
   var $defineProperties = $Object.defineProperties;
   var $defineProperty = $Object.defineProperty;
```

Comment:
`$defineProperties` is not used. I assume that you just accidentally commented on the wrong line?